docs/pages/build/eas-json.md

```diff
@@ -201,14 +203,16 @@ The schema of a build profile for a managed iOS project looks like this:
   "workflow": "managed",
   "credentialsSource": "local" | "remote" | "auto", // default: "auto"
   "releaseChannel": string, // default: "default"
-  "distribution": "store" | "internal" // default: "store"
+  "distribution": "store" | "internal", // default: "store"
+  "autoIncrement": boolean | "version" | "buildNumber" // default: "buildNumber"
 }
 ```
 
 - `"workflow": "managed"` indicates that your project is a managed one.
 - `credentialsSource` defines the source of credentials for this build profile. If you want to provide your own `credentials.json` file, set this to `local` ([learn more on this here](/app-signing/local-credentials.md)). If you want to use the credentials EAS already has stored for you, choose `remote`. If you're not sure what to do, choose `auto` (this is the default option).
 - `releaseChannel` is the release channel for the `expo-updates` package ([Learn more about this](../distribution/release-channels.md)). If you do not specify a channel, your binary will pull releases from the `default` channel. If you do not use `expo-updates` in your project then this property will have no effect.
 - `distribution` is the flow of distributing your app. If you choose `internal` you'll be able to share your build URLs with anyone, and they will be able to install the builds to their devices straight from the Expo website. The default is `store` which means your build URLs won't be sharable. [Learn more Internal Distribution](internal-distribution.md).
+- `autoIncrement` controls how EAS CLI bumps your application build version. App Store uses two values from `Info.plist` to identify the app build: `CFBundleShortVersionString` and `CFBundleVersion`. `CFBundleShortVersionString` is the version visible to users, whereas `CFBundleVersion` defines the build number. The permutation of those needs to be unique. You can choose to bump either of them. When set to `version`, the patch of `CFBundleShortVersionString` is bumped (e.g. `1.2.3` -> `1.2.4`). When set to `buildNumber`, `CFBundleVersion` the last component of the version is bumped (e.g. `1.2.3.39` -> `1.2.3.40`). Defaults to `false` - versions won't be bumped automatically.
```

Comment:
for managed description, we probably don't need to go into details, just which app.json fields will be bumped and links to app config docs. If you want to keep the same description I would also consider doing that for generic.